<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChooseEntityDialog</class>
 <widget class="QDialog" name="ChooseEntityDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>283</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Choose Entity</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="mainVLayout">
     <item>
      <layout class="QHBoxLayout" name="listContainerHLayout" stretch="0,0">
       <item>
        <layout class="QVBoxLayout" name="categoryVLayout">
         <item>
          <widget class="QLabel" name="categoryLabel">
           <property name="text">
            <string>Category</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="categoryListWidget"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="templateVLayout">
         <item>
          <widget class="QLabel" name="templateLabel">
           <property name="text">
            <string>Template</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="templateListWidget">
           <property name="selectionMode">
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="placementCheckBox">
       <property name="text">
        <string>Place relative to camera</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="cancelPushButton">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="importPushButton">
         <property name="text">
          <string>Import</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
